packages/dev/src/events/DevEvent.ts

```diff
@@ -104,15 +121,22 @@ class DevEventEmitter<T extends ServerDevEvent | ClientDevEvent> {
     res.write("retry: 2000\n")
     res.write("data: " + JSON.stringify(connectedEvent) + "\n\n")
 
-    this.emit(connectedEvent)
+    this.emitToListeners(connectedEvent, [res])
     if (this.lastEvent) {
```

Comment:
Possibly, but redundant sent events should be considered anyway by the listener, because there're other cases where an event is already received. I don't think this is an issue, or I'm not getting what you're hinting at 😄

Reply:
Ah, makes sense 👍